book/src/examples.md

```diff
@@ -41,29 +50,32 @@ Date values:
 A ZuKYC Pod exposes a single custom statement with one custom deduction rule.
 
 ```
-statement loan_check(receiver: PubKey):
-   - OR():
-      - AND(gov_id: Pod, paystub: Pod):
-         - eq(gov_id.pk, receiver)
-         # Not in the sanction list
-         - does_not_contain(SANCTION_LIST, gov_id.pk)
-         # Valid government-issued ID
-         - eq(gov_id.signer, ZOO_GOV)
-         - eq(gov_id.type, SIGNATURE)
-         # At least 18 years old
-         - lt(gov_id.date_of_birth, NOW_MINUS_18Y) # date_of_birdth is more than 18y old
-         - eq(paystub.signer, ZOO_DEEL)
-         - eq(paystub.type, SIGNATURE)
-         - eq(paystub.ssn, gov_id.ssn)
-         # At least one year of consistent employment with your current employer
-         - lt(paystub.start_date, NOW_MINUS_1Y) # start_date is more than 1y old
-         - gt(paystub.issue_date, NOW_MINUS_7D) # issue_date is less than 7d old
-         # Annual salary is at least $20,000
-         - gt(paystub.annual_salary, 20000)
-         # Private key knowledge
-         - hash(0, sk, gov_id.pk)
-         # Nullifier
-         - hash("ZooKyc", sk, nullifier)
+// receiver: PubKey, gov_id, paystub, sk_pok: Pod, nullifier, sk: Raw
+loan_check(receiver, private: gov_id, paystub, nullifier, sk, sk_pok) = AND(
+    Equal(?gov_id["pk"], receiver)
+    // Not in the sanction list
+    SetNotContains(SANCTION_LIST, receiver)
+    // Valid government-issued ID
+    Equal(?gov_id[KEY_SIGNER], ZOO_GOV)
+    Equal(?gov_id[KEY_TYPE], SIGNATURE)
+    // At least 18 years old
+    Lt(?gov_id.date_of_birth, NOW_MINUS_18Y) # date_of_birdth is more than 18y old
+    Equal(?paystub[KEY_SIGNER], ZOO_DEEL)
+    Equal(?paystub[KEY_TYPE], SIGNATURE)
+    Equal(?paystub[ssn], ?gov_id.ssn)
+    // At least one year of consistent employment with your current employer
+    Lt(?paystub["start_date"], NOW_MINUS_1Y) # start_date is more than 1y old
+    Gt(?paystub["issue_date"], NOW_MINUS_7D) # issue_date is less than 7d old
+    // Annual salary is at least $20,000
+    Gt(?paystub["annual_salary"], 20000)
+    // Private key knowledge
```

Comment:
I've updated this to proof knowledge of the sk by having a signed pod with an auth entry.